packages/open-next/src/build.ts

```diff
@@ -32,8 +33,7 @@ export async function build(
   const openNextDistDir = url.fileURLToPath(new URL(".", import.meta.url));
 
   const { config, buildDir } = await compileOpenNextConfig(
-    baseDir,
-    openNextConfigPath,
+    path.join(baseDir, openNextConfigPath ?? "open-next.config.ts"),
```

Comment:
@james-elicx `baseDir` is retrieved above on l 32:

```ts
const baseDir = process.cwd();
```

This means that `compileOpenNextConfig` is always called with an absolute path (`path.join(baseDir, openNextConfigPath ?? "open-next.config.ts"`).

So I think we shuold enforce and document that in `compileOpenNextConfig`.

Reply:
updated the jsdoc comment